gstd/src/common/mod.rs

```diff
@@ -19,6 +19,6 @@
 //! Common modules for each Gear smart contract.
 
 pub mod errors;
-#[cfg(target_arch = "wasm32")]
+#[cfg(feature = "panic-handler")]
```

Comment:
that way `gstd = { version = "*", default-features = false, features = ["com-handler"] }` is no-op due to lack of panic handler feature

Reply:
ok i did
```rust
#[cfg(feature = "panic-handler")]
pub mod panic_handler { }
```